docs/modules/operation/pages/events/sources/snmp-traps.adoc

```diff
@@ -2,23 +2,350 @@
+         c3800SysTrapSeverity&#38lt;/b&#38gt;&#38lt;/td&#38gt;&#38lt;td&#38gt;%parm[#3]%
+         &#38lt;/td&#38gt;&#38lt;td&#38gt;&#38lt;p;&#38gt;
+         clear(1) minor(2) major(3)&#38lt;/p&#38gt;
+         &#38lt;/td;&#38gt;&#38lt;/tr&#38gt;&#38lt;tr&#38gt;&#38lt;td&#38gt;&#38lt;b&#38gt;
+         c3800SysAggregateStatus&#38lt;/b&#38gt;&#38lt;/td&#38gt;&#38lt;td&#38gt;%parm[#4]%
+         &#38lt;/td&#38gt;&#38lt;td&#38gt;&#38lt;p;&#38gt;
+         clear(1) minor(2) major(3)</p>
+         &#38lt;/td;&#38gt;&#38lt;/tr&#38gt;&#38lt;/table&#38gt;
+  </descr>
+  <logmsg dest='logndisplay'><p>Cisco Event: C3900: Node Status has changed.</p></logmsg>
+  <severity>Indeterminate</severity>
+</event>
+----
+
+This event will match an SNMP trap whose enterprise OID (id) is equal to ".1.3.6.1.4.1.9.9.70.2", its generic trap value is enterprise specific (6), and the vendor trap value is 17.
+
+//I cannot find a MIB for this, so may need to delete this text. This is an old device, but updating this example would probably be difficult.
+The MIB for the C3800 device shows that the generic value 6 and the specific value of 17 for ciscoC3800SysAggregateStatusChange are X and Y respectively.
+So the event ...
+//
+
+Possible <mename> values include the following:
+
+* uei
+* source
+* host
+* snmphost
+* nodeid
+* interface
+* service
+* id
+* specific (vendor-specific error conditions and error codes. Refer to the associated MIB for details.)
+* generic (fixed, universal SNMP trap types)
+** Coldstart or warmstart
+** Linkup or linkdown
+** Authentication fails
+** egpNeighborloss (Agent cannot communicate with its EGP (Exterior Gateway Protocol) peer.)
+* community (community string in an SNMP trap)
+
+Note that the following elements are deprecated: `forward` and `filters`.
+
+
+You can use the `%` symbol as a wildcard in the mask values.
+For example, to match all Cisco events, you could use the following:
+
+[source, xml]
+----
+<mask>
+  <maskelement>
+    <mename>id</mename>
+    <mevalue>.1.3.6.1.4.1.9.%</mevalue>
+  </maskelement>
+</mask>
+----
+
+Note: The order in which events are listed in the eventconf.xml file is extremely important.
+The search will stop with the first event definition that matches the given event (see xref:operation:events/event-configuration.adoc[eventconf.xml]).
+
+Thus if the code with the wildcard was listed before the more specific ciscoC3800SysAggregateStatusChange event, the latter event would never be generated.
+
+Also note that the wildcard is simply a substring match.
+If a Cisco device generates an event with the enterprise OID of ".1.3.6.1.4.1.9" it would not match this event, as there is no trailing ".".
+If the trailing "." is left off, you must take care so that a trap with an OID of ".1.3.6.1.4.1.99" is listed before the ".1.3.6.1.4.1.9%" event or else it will match the more generic event.
```

Comment:
```suggestion
If a Cisco device generates an event with the enterprise OID of `.1.3.6.1.4.1.9` it would not match this event, as there is no trailing ".".
If the trailing "." is left off, you must take care so that a trap with an OID of `.1.3.6.1.4.1.99` is listed before the `.1.3.6.1.4.1.9%` event or else it will match the more generic event.
```